### PR TITLE
add turkish as a language in locale

### PR DESCRIPTION
**Pull request by alperen96**

### CODE REVIEW

File: locale/tr_TR.UTF-8/LC_MESSAGES/palma.po [hunk 1]
```diff
@@ -0,0 +1,515 @@
+# PalMA translation.
+# Copyright (C) 2017 Universitätsbibliothek Mannheim
+# This file is distributed under the same license as the PalMA package.
+# Stefan Weil <stefan.weil@bib.uni-mannheim.de>, 2014.
+# Dennis Müller <dennis.mueller@bib.uni-mannheim.de>, 2017.
```
Comment (stweil) [inline locale/tr_TR.UTF-8/LC_MESSAGES/palma.po]: Here you may replace my name and Dennis Müller by your own name and e-mail address.

File: locale/tr_TR.UTF-8/LC_MESSAGES/palma.po [hunk 2]
```diff
@@ -0,0 +1,515 @@
...
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: PalMA 0.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2018-11-19 20:33+0100\n"
+"PO-Revision-Date: 2017-09-12 14:38+0200\n"
+"Last-Translator: Dennis Müller <dennis.mueller@bib.uni-mannheim.de>\n"
```
Comment (stweil) [inline locale/tr_TR.UTF-8/LC_MESSAGES/palma.po]: Replace that using your name.

File: locale/tr_TR.UTF-8/LC_MESSAGES/palma.po [hunk 3]
```diff
@@ -0,0 +1,515 @@
...
+"Language-Team: English <en@li.org>\n"
```
Comment (stweil) [inline locale/tr_TR.UTF-8/LC_MESSAGES/palma.po]: Replace `English` by `Turkish`.

File: locale/tr_TR.UTF-8/LC_MESSAGES/palma.po [hunk 4]
```diff
@@ -0,0 +1,515 @@
...
+msgstr ""
+
+#: index.php:223
+#, fuzzy
+msgid "Enter valid URL"
+msgstr "Please enter URL"
+
+#: index.php:563
+msgid "You were disconnected!"
+msgstr ""
```
Comment (stweil) [inline locale/tr_TR.UTF-8/LC_MESSAGES/palma.po]: `msgstr` should give the Turkish text for all messages. Without that there will not be a Turkish user interface.

**Comment by jmechnich** [comment]
Closing this as the translations are actually missing.

**Pull request closed without merging**